lms/djangoapps/support/static/support/jsx/entitlements/data/reducers/form.js

```diff
@@ -0,0 +1,24 @@
+import { formActions, entitlementActions } from '../actions/constants';
+
+const clearForm = state => ({
+  ...state,
+  isOpen: false,
+  activeEntitlement: null,
+});
+
+const form = (state = {}, action) => {
```

Comment:
Should the initial state be an empty object or 

```javascript
{ isOpen: false, activeEntitlement: null }
```

Reply:
I think this is fine, (this summary)[https://redux.js.org/recipes/structuring-reducers/initializing-state#summary] looks like the preloaded state from createStore will override the starting state in the reducer.